docker/Dockerfile.dev

```diff
@@ -0,0 +1,14 @@
+FROM python:3.7
+
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONDONTWRITEBYTECODE=1
+ENV PATH "/root/.poetry/bin:$PATH"
+RUN curl -sSL https://raw.githubusercontent.com/sdispater/poetry/master/get-poetry.py | python
+
+WORKDIR /srv/search
```

Comment:
We've used just `/app` elsewhere rather than copying this format (which comes from eLife's VMs).

Reply:
updated to use `/srv/app`